server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -43,7 +43,6 @@
  * released version, they can use "0.10.0" when upgrading to the 0.10.0 release.
  */
 public enum MetadataVersion {
-    UNINITIALIZED(-1, "0.0", ""),
```

Comment:
nit: There is a comment below about omitting UNINITIALIZED that we need to update.